2019 midyear update (#1068)
writethedocs/www: docs/blog/write-the-docs-mid-year-update-2019.rst

.. post:: Sep 5, 2019
   :tags: stats, newsletter
   :author: Eric Holscher

Write the Docs 2019 Community Update
====================================

Hey there documentarians.
The newsletter team is taking the month off,
but that doesn't mean the community is standing still.
We decided to send out a community update to let you know what's going on with the community. 

Salary Survey
-------------

> The biggest thing that we're excited about is a new :doc:`Salary Survey </surveys/salary-survey-sep-2019>`.

Author: ```suggestion
The thing that we're most excited about is our new :doc:`Salary Survey </surveys/salary-survey-sep-2019>`.
```

> Our goal is to help people understand the pay scales and benefits in the software industry around documentation.

Author: ```suggestion
Our goal is to get a clear idea of the pay scales and benefits in the software industry around documentation.
```

> This will help people get a better understanding of what an appropriate salary is,

Author: ```suggestion
Publishing the results will help people get a better understanding of what an appropriate salary is,
```

> and then negotiate for a better one.

Author: ```suggestion
and negotiate a better one.
```

The survey is 21 questions broken up into 4 parts.
Please fill it out here: https://www.surveymonkey.com/r/wtd-salary-2019

The salary will be open until October 15th, 2019,

> after that we will process the data and publish the results.

Author: ```suggestion
after that we will process the data and publish the anonymized results.
```

Community Interviews
--------------------

We have finished up the summer internship that we ran this year with `Mohit Patel`_.
We now have `13 interviews with people in the community`_ sharing their history in the industry,
as well as suggestions for folks who are just getting started.

We're excited to have this resource available for people who are hoping to get into the industry.
It's often hard to understand how to get started,
and having so many different stories and backgrounds really shows there isn't a "proper" path,
but many different ways.

.. _Mohit Patel:  https://mohitpatel.design/2019/08/13/write-the-docs-internship-retrospective/
.. _13 interviews with people in the community: http://www.writethedocs.org/hiring-guide/#community-spotlight

A US East Conference
--------------------

We're also excited to start planning another US conference in 2020.

> The goal will be to make closer to folks on the east coast,

Author: ```suggestion
We want to make this one somewhere closer to folks on the east coast,
```

> but we haven't figured out where to host it yet.

Author: ```suggestion
but we haven't figured out exactly where to host it yet.
```

Currently we're in the process of trying to find suitable venues.
We have more details in our :doc:`request for venues </conf/cfp/east-2020>`.
If you know of a good venue that you think would work well for a Write the Docs conference,
please let us know!

Meetups
-------

We're almost at 50 meetups now, including a newly created `remote meetup <http://writethedocsremote.tk/#events/>`_!

> Kick off event 09/10/2019 8:30pm EDT.

Author: ```suggestion
The kick-off event was a few days ago, and we're eager to hear how it went.
```
